stable/kibana/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: "docker.elastic.co/kibana/kibana-oss"
-  tag: "6.7.0"
+  tag: "7.9.3"
```

Comment:
should this be `7.10.1` (what `appVersion` is set to)?
https://github.com/mesosphere/charts/pull/933/files#diff-38a0491093d5cf5568e8c1f59ecd4669b929d03e8c476864bcddf31e6d4abf46R4

Reply:
No, we cannot upgrade to 7.10.1 unfortunately, the plugin for it is currently broken, we can change this when those problems have been resolved.

Reply:
ok got it thanks, let's update the appVersion then